docs/reference/src/main/asciidoc/gettingstarted.asciidoc

```diff
@@ -20,8 +20,8 @@ following:
 
 * the latest release of Weld, which contains the examples
 * Maven 3, to build and deploy the examples
-* a supported runtime environment (minimum versions shown)
-** WildFly 8.0.0.Final,
+* optionally a supported runtime environment (minimum versions shown)
+** WildFly 8.0.0.Final (10.1.0.Final recommended),
```

Comment:
This doc will concern running Weld 3.x on WFLY - I am pretty sure you cannot run the numberguess as easily as doing `mvn wildfly:run`. And furthermore, not on WFLY 8 for sure. Or am I missing something obvious?